### DISCUSSION TITLE
adjust spatie laravel-google-calendar package to work with env variables

### DISCUSSION THREAD

**Discussion by Badlapje** [spatie/laravel-google-calendar#226] [Ideas]
Make an adjustment so the spatie package works with env variables rather than with a json file.

**Comment by Badlapje** [comment]
hum, seem to have made this issue in the wrong repo.  Wanted to make one in our repo so i can make a PR for this.  Apologies!

**Reply by mkokio** [reply]
Is it possible now for the spatie package to work with env variables?
